pegasus/sites.v3/code.org/public/js/admins_email_signup_form.js

```diff
@@ -0,0 +1,17 @@
+// Code provided from Google reCAPTCHA
+// https://www.sfdcpoint.com/salesforce/salesforce-web-to-lead-form-with-recaptcha/
+function timestamp() { var response = document.getElementById("g-recaptcha-response"); if (response == null || response.value.trim() == "") {var elems = JSON.parse(document.getElementsByName("captcha_settings")[0].value);elems["ts"] = JSON.stringify(new Date().getTime());document.getElementsByName("captcha_settings")[0].value = JSON.stringify(elems); } } setInterval(timestamp, 500);
```

Comment:
This looks like it's wrapped weirdly which makes it hard to parse. Not blocking, but could you add line breaks and indentation here?

Reply:
I'm getting lint errors here –– not sure this will pass drone?

Reply:
@kelbyhawn to continue from our previous discussion, these lint errors are part of what we gain by extracting the js 🎉  apologies that they are extra work (especially when they just complain about formatting), but the benefit is that they can catch typos and other errors in our js

Reply:
to make sure you see the lint errors in your local dev environment, make sure you have installed the "hooks" as described in https://github.com/code-dot-org/code-dot-org/blob/staging/SETUP.md#overview

Reply:
Thanks for adding this context, Dave! Forgot, Kelby, you may not have linting set up –– reach out if you have questions about this.

Reply:
Thanks @davidsbailey and @megcrenshaw! I do have linting set up 🙌 

I just reformatted w/ the line breaks, so the only error I'm getting now is `21:10  error  'recaptchaCallback' is defined but never used  no-unused-vars`, which is related to this comment: https://github.com/code-dot-org/code-dot-org/pull/50272#discussion_r1106297968 — any thoughts on what I should do here?

Reply:
Ah yes. Gnarly. Our linter doesn't pick up that it's being used in an HTML file!

So you can do this
```
// eslint-disable-next-line no-unused-vars
function recaptchaCallback() {
```

Simpler would be
```
function recaptchaCallback() { // eslint-disable-line no-unused-vars
```
but then it gets mad about line length being too long. What a time to be alive.

But at least we're still specific about (a) the error to disable (no-unused-vars) and (b) the exact line

Reply:
That worked, thanks! Just updated 🎉 

(I lol'd at "What a time to be alive." 😂)